engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/impl/web/bootstrap/RestContainerBootstrap.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.rest.impl.web.bootstrap;
+
+import static org.camunda.bpm.engine.rest.util.EngineUtil.getProcessEngineProvider;
+import static org.camunda.bpm.engine.rest.util.EngineUtil.lookupProcessEngine;
+
+import java.util.Set;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+
+import org.camunda.bpm.engine.ProcessEngine;
+
+public class RestContainerBootstrap implements ServletContextListener {
+
+  @Override
+  public void contextInitialized(ServletContextEvent sce) {
+    String serverInfo = sce.getServletContext().getServerInfo();
+    if (serverInfo != null && !sce.getServletContext().getServerInfo().isEmpty() ) {
+      Set<String> processEngineNames = getProcessEngineProvider().getProcessEngineNames();
+      for (String engineName : processEngineNames) {
+        ProcessEngine processEngine = lookupProcessEngine(engineName);
+        try {
+          if (processEngine.getProcessEngineConfiguration().getTelemetryRegistry() != null &&
+              processEngine.getProcessEngineConfiguration().getTelemetryRegistry().getApplicationServer() == null) {
+
+            processEngine.getProcessEngineConfiguration()
+                .getTelemetryRegistry()
+                .setApplicationServer(serverInfo);
+          }
+        } catch (Exception e) {
+          // do nothing
+        }
+      }
+    }
```

Comment:
```suggestion
    if (serverInfo != null && !serverInfo.isEmpty() ) {
      Set<String> processEngineNames = getProcessEngineProvider().getProcessEngineNames();
      for (String engineName : processEngineNames) {
        ProcessEngine processEngine = lookupProcessEngine(engineName);
        ProcessEngineConfiguration configuration = processEngine.getProcessEngineConfiguration();
        TelemetryRegistry telemetryRegistry = configuration.getTelemetryRegistry();
        if (telemetryRegistry != null && telemetryRegistry.getApplicationServer() == null) {
            telemetryRegistry.setApplicationServer(serverInfo);
        }
      }
    }
```

Reply:
Instead of this exception handling, let's just take care that in `ParseUtil#parseServerVendor` no exceptions are thrown.